controllers/notifications.js

```diff
@@ -0,0 +1,51 @@
+const notificationQuery = require('../models/notifications')
+/**
+ * Fetches the notifications data for current user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const getNotificationsForUser = async (req, res) => {
+  try {
+    let { page: currentPage, n: limit } = req.query
+    const { id: userId } = req.userData
+
+    if (!currentPage) {
+      currentPage = 1
+    }
+
+    if (!limit) {
+      limit = 10
+    }
+
+    let parsedCurrentPage = +currentPage
+    let parsedLimit = +limit
+
+    if (isNaN(parsedCurrentPage) || parsedCurrentPage <= 0) {
+      parsedCurrentPage = 1
+    }
+
+    if (isNaN(parsedLimit) || parsedLimit <= 0) {
+      parsedLimit = 10
+    }
```

Comment:
NIT: Try ternary operators

Reply:
But by using **ternary operators*, won't it affect the code readability, since I need to use the ***nested*** ternary operators.

Reply:
Well, it's up to you, whether use it or not, If you feel it would be hard to read then you can skip it :smiley:

Reply:
But why would you need nested ternary operators?
This would be some thing like this 
`const parsedCurrentPage = (isNaN(parsedCurrentPage) || parsedCurrentPage <= 0) ? 1 : currentPAge`, right or am I missing something?

Reply:
Yes , correct